### PR TITLE
ubuntu_application_sensors: Adapt to Android 7.1 API changes

### PR DESCRIPTION
**Pull request by fredldotme**
Use android::SensorManager::getInstanceForPackage() instead of android::SensorManager::getInstance() and set the event loop name.

### CODE REVIEW

**Comment by APokorny** [comment]
I think you need to enclose the changes in ANDROID_VERSION_MAJOR >= 7 blocks.
Because we will probably build ubp-5.1 system images with that code.

**Comment by fredldotme** [comment]
> I think you need to enclose the changes in ANDROID_VERSION_MAJOR >= 7 blocks.
> Because we will probably build ubp-5.1 system images with that code.

Agreed, should build now with 5.1 based device abstractions.

**Comment by fredldotme** [comment]
Note: Build failures occur due to installing missing packages via apt

```
Err http://repo.ubports.com PR-3/main arm64 liburl-dispatcher1-dev arm64 0.2+ubports+0~20180611143914.3~1.gbp9a4659

  404  Not Found [IP: 2606:4700:20::6818:9 80]
```

Also, the changes are Android-side, which isn't handled by the build bot.

**Comment by APokorny** [comment]
Yes you can ignore the build error. It is not related to the change. There seems to be something wrong with the o way the armhf build selects the repository fetch the url-dispatcher package from

File: android/hybris/ubuntu_application_sensors_for_hybris.cpp [hunk 1]
```diff
@@ -77,9 +77,25 @@ static BackwardSensorTypeLut init_backward_sensor_type_lut()
     return lut;
 }
 
+#ifdef ANDROID_VERSION_MAJOR >= 7
```
Comment (APokorny) [inline android/hybris/ubuntu_application_sensors_for_hybris.cpp]: #if ANDROID_VERSION_MAJOR >= 7

Comment (APokorny) [reply android/hybris/ubuntu_application_sensors_for_hybris.cpp]: and same below ..

**Review by APokorny — CHANGES_REQUESTED** [review]

File: android/hybris/ubuntu_application_sensors_for_hybris.cpp (hunk 1 above)
Comment (fredldotme) [reply android/hybris/ubuntu_application_sensors_for_hybris.cpp]: alcohol

**Review by APokorny — APPROVED** [review]

**Pull request merged**